upgrade-guides/v4.0-v5.0.md

```diff
@@ -0,0 +1,37 @@
+# How to Upgrade Hazelcast-wm Version 4.0 to Version 5.0
+
+This guides summarizes what you need to know if you are upgrading from Hazelcast-wm 4.0 version to 5.0 version.
+
+
+## Hazelcast-wm supports Hazelcast 5.x
+
+Hazelcast-wm supported Hazelcast 4.x versions, whereas Hazelcast-wm supports Hazelcast 5.x. One notable change in Hazelcast 5.x 
+that can affect Hazelcast-wm usage is that Hazelcast clients have indefinite cluster connect timeout by default,
+which was previously 20 seconds. For more info on the topic, see
+[this page](https://docs.hazelcast.com/hazelcast/5.4/clients/java#configuring-client-connection-retry)
```

Comment:
Which version(s) of hazelcast-wm supported Hazelcast 4.x? This first sentence doesn't make sense without clarification. e.g.

```suggestion
Previous versions of hazelcast-wm supported Hazelcast Platform 4.x versions; to use Hazelcast Platform 5. you need to upgrade your version of hazelcast-wm. 
```

```suggestion
In Hazelcast Platform 5.x, Hazelcast clients now default to indefinite cluster connect timeout where previously this timeout was 20 seconds. This is important for hazelcast-wm usage - for more info, see
[this page](https://docs.hazelcast.com/hazelcast/5.4/clients/java#configuring-client-connection-retry) or follow the guidance below to restore this setting.
```

Reply:
applied both with slight diff in the first suggestion, I added v4.0 explicitly